portals/devportal/src/main/webapp/source/src/app/components/Apis/Details/ApiChat/components/ApiChatPoweredBy.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { FormattedMessage } from 'react-intl';
+import KeyboardBackspaceIcon from '@mui/icons-material/KeyboardBackspace';
+import SettingsOutlinedIcon from '@mui/icons-material/SettingsOutlined';
+import { styled } from '@mui/material/styles';
+
+const PREFIX = 'ApiChatPoweredBy';
+
+const classes = {
+    poweredBy: `${PREFIX}-poweredBy`,
+};
+
+const Root = styled('div')(() => ({
+    [`& .${classes.poweredBy}`]: {
```

Comment:
Since this only has one simple css class, we can use the `sx` prop instead to apply the css itself.